packages/material-ui/src/useAutocomplete/useAutocomplete.d.ts

```diff
@@ -0,0 +1,16 @@
+export {
+  useAutocomplete as default,
+  AutocompleteChangeDetails,
+  AutocompleteChangeReason,
+  AutocompleteCloseReason,
+  AutocompleteFreeSoloValueMapping,
+  AutocompleteGetTagProps,
+  AutocompleteGroupedOption,
+  AutocompleteHighlightChangeReason,
+  AutocompleteInputChangeReason,
+  AutocompleteValue as Value,
+  createFilterOptions,
+  CreateFilterOptionsConfig,
+  FilterOptionsState,
+  UseAutocompleteProps,
+} from '@material-ui/unstyled/AutocompleteUnstyled';
```

Comment:
Why not export * instead of listing each module?

Reply:
I don't want to export `Value`. I renamed it to `AutocompleteValue` as this is a type specific to Autocomplete.

Reply:
Oh, I thought that we had already prefixed everything, well spotted.